packages/acceptance-tests/pkg-tests-specs/sources/commands/config.test.js

```diff
@@ -0,0 +1,108 @@
+  [`folder with rcfile`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
+  },
+  [`folder with rcfile without trailing newline`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test`);
+  },
+  [`folder with rcfile and rc in parent`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
+    await writeFile(`${path}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
+  },
+  [`folder with rcfile and rc in ancestor parent`]: async path => {
+    await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
+    await writeFile(`${path}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
+  },
+};
+
+function cleanupPlainOutput(output, path) {
+  return output
+      // replace the generated workspace root with a constant
+      .replace(new RegExp(path, `g`), FAKE_WORKSPACE_ROOT)
+      // replace the generated registry server URL with a constant
+      .replace(/^(\s*npmRegistryServer.*?)(['"]?)http:\/\/localhost:\d+\2\s*$/mg, `$1$2${FAKE_REGISTRY_URL}$2`);
+}
+
+function cleanupJsonOutput(output, path) {
+  let outputObject;
+
+  try {
+    outputObject = JSON.parse(output);
+  } catch {
+    return cleanupPlainOutput(output, path);
+  }
+
+  // the default globalFolder contains the user's home folder, override that value
+  outputObject[`globalFolder`].default = `DEFAULT_GLOBAL_FOLDER`;
+  
+  // replace the generated registry server URL with a constant
+  outputObject[`npmRegistryServer`].effective = FAKE_REGISTRY_URL;
+
+  const pathRegExp = new RegExp(path, `g`);
+
+  for (const setting of Object.values(outputObject)) {
+    if (typeof setting.source === 'string')
+      setting.source = setting.source.replace(pathRegExp, FAKE_WORKSPACE_ROOT);
+
+    if (typeof setting.default === 'string')
+      setting.default = setting.default.replace(pathRegExp, FAKE_WORKSPACE_ROOT);
+
+    if (typeof setting.effective === 'string')
+      setting.effective = setting.effective.replace(pathRegExp, FAKE_WORKSPACE_ROOT);
+  }
+
+  return JSON.stringify(outputObject);
+}
+
+const options = {
+  [`without flags`]: {cleanupStdout: cleanupPlainOutput, flags: []},
+  [`showing the source`]: {cleanupStdout: cleanupPlainOutput, flags: [`--why`]},
+  [`showing explanation`]: {cleanupStdout: cleanupPlainOutput, flags: [`--verbose`]},
+  [`as json`]: {cleanupStdout: cleanupJsonOutput, flags: [`--json`]},
+};
+
+describe(`Commands`, () => {
+  describe(`config`, () => {
+    for (const [environmentDescription, environment] of Object.entries(environments)) {
+      for (const [optionDescription, {flags, cleanupStdout}] of Object.entries(options)) {
+        // Note: Don't use concurrent testing w/ snapshots, they don't work together
```

Comment:
Concurrent testing in general is completely broken in Jest and should be avoided (all the tests are spawned at definition time, regardless of beforeEach, afterEach, skip, only, etc ...).

Reply:
Copy paste leftover from the constraints spec file. I'll remove both